packages/core/src/App/Components/Elements/Errors/error-component.jsx

```diff
@@ -37,6 +38,7 @@ const ErrorComponent = ({
 
     const refresh_message = should_show_refresh ? localize('Please refresh this page to continue.') : '';
     const hide_notification = localStorage.getItem('hide_close_mx_mlt_account_notification');
+    const bot_error_button = window.location.pathname === routes.bot && !redirect_label && [];
```

Comment:
It's not recommended to add bot-specific logic in the core component.

Reply:
@balakrishna-deriv refactored ✅